### ISSUE TITLE
Same Dates displayed as two different dates

### ISSUE THREAD

**Issue by kyawzazaw** [rarfell/dimeApp#19]
![image](https://github.com/rarfell/dimeApp/assets/10335802/b1ed15f4-0395-4452-b210-1b2e6b3a5fca)

**Comment by kyawzazaw** [comment]
Sorry, I thought I use bug as label

**Comment by izyumidev** [comment]
Could you provide ways to reproduce this? Is there anything specific you noticed? Did this just happen recently, or is this a recurring issue? Thank you for reporting on this issue.

**Comment by kyawzazaw** [comment]
I am not exactly sure on how to reproduce.

I think this happens if I retroactively act an expense much later.

![image](https://github.com/rarfell/dimeApp/assets/10335802/a47e93a6-a79c-463b-898f-17c266e9dd4e)

**Comment by izyumidev** [comment]
Thank you for sharing this. We will take a look into this issue.

**Comment by rarfell** [comment]
Hey @kyawzazaw, thanks for highlighting this issue! It is a known bug caused by logging transactions in different timezones (the transactions are grouped according to the start of day of the timezone it was logged in). We're looking into different ways to address this issue, and will keep you posted! Thank you for bearing with this 😅

**Comment by jaimemiranda1** [comment]
Yes, as @rarfell said, this happens whenever you add transactions on the same day from different timezones. This bug has been in the app since v1.0.
